Overlays/Windows/Overlays/D3D12/Runtime/WNGraphics/src/D3D12/WNDevice.cpp

```diff
@@ -40,6 +40,7 @@
 
 #include <d3d12.h>
 #include <dxgi1_4.h>
+#include <functional>
```

Comment:
What do we use from this header in here?

Reply:
bind comes from here. (2019 it doesnt get transitively included)